R/add-cr.R

```diff
@@ -0,0 +1,291 @@
+add_hazard_cr <- function(newdata, object, ci = TRUE, se_mult = 2,
```

Comment:
Same here, ideally just one user facing function, `add_hazard`. Is it possible? Or are there problems?